site/docs/pages/is-entrypoint.mdx

```diff
@@ -0,0 +1,34 @@
+# `isValidAAEntrypoint`
+
+The `isValidAAEntrypoint` utility is designed to verify if the entrypoint address is a valid v6 entrypoint.
+
+## Usage
+
+:::code-group
+
+```tsx [code]
+import { isValidAAEntrypoint } from '@coinbase/onchainkit';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
```

Comment:
In this example I think is fair to avoid showing this line.

So you could instead have something
```
const AAImplementationAddress = '0x123`; 
const isValid = isValidAAEntrypoint({ entrypoint: AAImplementationAddress });
```
to showcase what you are checking.

otherwise the example feels like
is true === to true :D

Reply:
Good point. Updated to:

import { isValidAAEntrypoint } from '@coinbase/onchainkit';

```
const AAImplementationAddress = '0x123`;
const isValid = isValidAAEntrypoint({ entrypoint: AAImplementationAddress });

if (isValid) {
  console.log('The entrypoint is valid.');
} else {
  console.log('Invalid entrypoint.');
}
```